src/components/KuadrantPoliciesPage.tsx

```diff
@@ -38,6 +37,7 @@ import {
   useActivePerspective,
 } from '@openshift-console/dynamic-plugin-sdk';
 import './kuadrant.css';
+import getModelFromResource from '../utils/getModelFromResource';
```

Comment:
not a comment for this particular line, just that this isn't visible in the PR:


is this hooked up to the Create DNSPolicy button on the policies view?